mappings/net/minecraft/util/io/PacketByteBuf.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2540 net/minecraft/util/PacketByteBuf
+CLASS net/minecraft/class_2540 net/minecraft/util/io/PacketByteBuf
```

Comment:
This is more like networking than IO

Reply:
Yeah, but then it would be the only class in that package. IO is a superset of networking anyway.

Reply:
Nope, networking is not io. move this class to `net/minecraft/network` if you really want to make things right